src/main.js

```diff
@@ -126,8 +108,18 @@ var Meyda = {
         previousSignal:this.previousSignal,
         previousAmpSpectrum:this.previousAmpSpectrum,
         previousComplexSpectrum:this.previousComplexSpectrum,
+        numberOfMFCCCoefficients:this.numberOfMFCCCoefficients,
       });
-    }    else {
+    };
+
+    if (typeof feature === 'object') {
+      feature.reduce((acc, el) =>
+        Object.assign({}, acc, {
+          [el]: extract(el),
+        }), {});
```

Comment:
good to see `Object.assign`! Might be worth to check if we're claiming to support any Node versions or browsers which don't support it – although I doubt we do.

Reply:
I would've thought babel would just deal with that? I was using `Object.assign` as an alternative to the object spread operator which isn't in babel env defaults yet.

Reply:
[Here](http://kangax.github.io/compat-table/es6/#test-Object_static_methods_Object.assign)'s the compat table for Object.assign, I'll go through it when I have time :)

Reply:
well normally you have to include `babel-polyfill` to support ES features which aren't just syntax but actual APIs (that includes syntactic features the spread operator for instance, because under the hood you need iterables etc.) – we obviously can't bundle `babel-polyfill`, it's massive!

I can't really find anywhere in the docs where we specify compatibility – but looking at the `.assign` compat table I think we're safe, in browsers we're only targetting Web Audio-enabled ones so IE doesn't matter, and Node supported this method since at least v4.

We could literally just say we support node 4+ and it's sorted 😄 I'd even go as far as only 8+ since that is the current LTS release and it even has ES2016 features which we could then happily use.

Reply:
We already say we support node 4+ 🎉